server/routes/alumni.js

```diff
@@ -41,7 +41,8 @@ router.post('/', async (req, res, next) => {
                 //posts: [{type: Schema.Types.ObjectId, ref: 'postSchema'}]
                 availabilities: availabilities,
                 timeZone: timeZone,
-                zoomLink: zoomLink
+                zoomLink: zoomLink,
+                verified: true
```

Comment:
I see, it kind of does make sense to move the `approved` from user collection to `Alumni` collection

Reply:
We can discuss this more during standup